src/sentry/static/sentry/app/routes.jsx

```diff
@@ -486,9 +486,11 @@ function routes() {
       <Route
         path="auth/"
         name="Auth Providers"
-        componentPromise={() =>
-          import(/*webpackChunkName: OrganizationAuth*/ './views/settings/organizationAuth')}
-        component={errorHandler(LazyLoad)}
+        component={
+            HookStore.get('component:org-auth-view').length
+              ? HookStore.get('component:org-auth-view')[0]()
+              : OrganizationAuth
```

Comment:
This is not imported in the routes file because it was previously codesplit. Maybe we should re-think this so that we can support code splitting when using hooks like this.

Reply:
Yes- i'm game to try a new pattern out. What were you thinking?